src/jobservice/job/known_jobs.go

```diff
@@ -40,4 +40,29 @@ const (
 	SystemArtifactCleanupVendorType = "SYSTEM_ARTIFACT_CLEANUP"
 	// ScanDataExportVendorType : the name of the scan data export job
 	ScanDataExportVendorType = "SCAN_DATA_EXPORT"
+	// ExecSweepVendorType: the name of the execution sweep job
+	ExecSweepVendorType = "EXECUTION_SWEEP"
+	// ScanAllVendorType: the name of the scan all job
+	ScanAllVendorType = "SCAN_ALL"
 )
+
+var (
```

Comment:
Adjust the logic of cleaning up registration entries to a unified location for better management, maintenance and to avoid some pitfalls caused by init.